### PR TITLE
Rebuild plugin_codelab

### PR DESCRIPTION
**Pull request by domesticmouse**

### CODE REVIEW

**Comment by domesticmouse** [comment]
I'm unsure why `plugin_codelab` is failing only on the `ubuntu-latest` stable CI check, and not macOS nor Windows stable. I'm hoping https://github.com/flutter/codelabs/pull/57 fixes it.

**Comment by domesticmouse** [comment]
Related issue: https://github.com/flutter/flutter/issues/74207

**Comment by domesticmouse** [comment]
TBR @johnpryan

File: plugin_codelab/.idea/libraries/Dart_SDK.xml [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+      <root url="file:///home/brett/flutter/bin/cache/dart-sdk/lib/collection" />
+      <root url="file:///home/brett/flutter/bin/cache/dart-sdk/lib/convert" />
+      <root url="file:///home/brett/flutter/bin/cache/dart-sdk/lib/core" />
+      <root url="file:///home/brett/flutter/bin/cache/dart-sdk/lib/developer" />
+      <root url="file:///home/brett/flutter/bin/cache/dart-sdk/lib/html" />
+      <root url="file:///home/brett/flutter/bin/cache/dart-sdk/lib/io" />
+      <root url="file:///home/brett/flutter/bin/cache/dart-sdk/lib/isolate" />
+      <root url="file:///home/brett/flutter/bin/cache/dart-sdk/lib/math" />
+      <root url="file:///home/brett/flutter/bin/cache/dart-sdk/lib/mirrors" />
+      <root url="file:///home/brett/flutter/bin/cache/dart-sdk/lib/typed_data" />
```
Comment (johnpryan) [inline plugin_codelab/.idea/libraries/Dart_SDK.xml:14]: Delete this directory?

**Review by johnpryan — COMMENTED** [review]
LGTM

File: plugin_codelab/.idea/libraries/Dart_SDK.xml (hunk 1 above)
Comment (domesticmouse) [reply plugin_codelab/.idea/libraries/Dart_SDK.xml:14]: I'm guessing I should .gitignore the .idea directory?

Comment (domesticmouse) [reply plugin_codelab/.idea/libraries/Dart_SDK.xml:14]: PTAL at https://github.com/flutter/codelabs/pull/60

**Pull request merged**